### DISCUSSION TITLE
docker6.5.0要求的账号密码是什么

### DISCUSSION THREAD

**Discussion by igmkgu** [jeessy2/ddns-go#1113] [General]
之前用6.3.3也没什么账号密码，直接就能用，升了6.5.0也没要求输入账号密码直接进了页面，但配置完点save就提示要账号密码，关键也没让我输入过，也没找到哪里输。

**Comment by jeessy2** [comment]
页面就那么多属性，多看下不就找到了。
禁止公网访问下方
参考#1098

**Reply by jeessy2** [reply]
为了兼容以前没设置密码的就自动登陆了。如果不自动登陆，在登陆页面，那你知道怎么输入密码不？
进入设置就可以了
